src/components/book.js

```diff
@@ -4,16 +4,22 @@ import DeleteBookButton from './deleteBookButton';
 
 const Book = ({ book, onDelete }) => (
   <div className="book">
-    <h3>{book.title}</h3>
+    <p>
+      Title:
+      {' '}
+      {book.title}
+    </p>
     <p>
       Author:
+      {' '}
       {book.author}
     </p>
     <p>
       Category:
+      {' '}
       {book.category}
     </p>
-    <DeleteBookButton onClick={onDelete} bookId={book.id} />
+    <DeleteBookButton onClick={onDelete} itemId={book.item_id} />
```

Comment:
- [ ] I can add books to the API, but I am unable to remove them. Kindly fix this. It may have to do with this error in the browser.

  ![image](https://github.com/George7h/bookstores/assets/54848363/ca4fe9f9-078e-44ac-9b89-4a2e0f9feccd)